model/src/main/kotlin/config/AdvisorConfiguration.kt

```diff
@@ -155,11 +155,11 @@ data class VulnerableCodeConfiguration(
     /**
      * The base URL of the VulnerableCode REST API. By default, the public VulnerableCode instance is used.
      */
-    val serverUrl: String?,
+    val serverUrl: String? = null,
 
     /**
      * The optional API key to use.
      */
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
-    val apiKey: String?
+    val apiKey: String? = null
```

Comment:
I believe the comment is outdated and this property actually is not optional anymore. @oheger-bosch, do you know if also self-hosted VulnerableCode instances require an API key by now?